doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -22,10 +22,10 @@
 
 ## 3.1 Introduction
 
-The CNTT Kubernetes Reference Architecture (RA) is intended to be an industry
+The Anuket Kubernetes Reference Architecture (RA) is intended to be an industry
 standard independent Kubernetes reference architecture that is not tied to any
 specific offering or distribution. No vendor-specific enhancements are required
-in order to achieve conformance to CNTT principles; conformance is achieved by
+in order to achieve conformance to Anuket principles; conformance is achieved by
```

Comment:
This, and a couple of others in this PR, have assumed the scope of CNTT and Anuket to be the same (i.e. that CNTT Principles == Anuket Principles.  Might there be other principles in Anuket that aren't included in the what-was CNTT "Common" documentation?  i.e. the principles defined in the [common documentation tree](https://cntt.readthedocs.io/en/latest/common/) - I assume they are relevant only to the former-CNTT projects?

Reply:
Thanks for the comment! We tried to clarify this, are you OK with the changes?

Reply:
Yes that sounds good, thanks!